### PR TITLE
adding digital gardening feature page

### PR DESCRIPTION
**Pull request by laurosilvacom**
links and adds the initial curation for the [digital gardening page](https://egghead-io-nextjs-git-ls-gardening-clubs-eggheadio1.vercel.app/learn/digital-gardening) using Sanity. 

design can be improved as we further curate content on this page 

image

doggo

### CODE REVIEW

File: src/components/pages/home/index.tsx [hunk 1]
```diff
@@ -114,34 +117,27 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
...
-                        <a href="https://joelhooks.com/digital-garden">
-                          Joel Hooks
-                        </a>
-                      </blockquote>
+                      <Markdown className="prose dark:prose-dark dark:prose-sm-dark prose-sm mt-4">
+                        {featureDigitalGardening.description}
+                      </Markdown>
+                      <Markdown className="prose dark:prose-dark dark:prose-sm-dark prose-sm mt-4">
+                        {featureDigitalGardening.quote.description}
+                      </Markdown>
```
Comment (laurosilvacom) [inline src/components/pages/home/index.tsx:134]: We are now using data from Sanity for the description and title.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -33,7 +33,7 @@ export default IndexPage
 
 const featureQuery = groq`
 {
-  'featureCallOut': ${digitalGardeningQuery}
+  'featureDigitalGardening': ${digitalGardeningQuery},
```
Comment (laurosilvacom) [inline src/pages/index.tsx:36]: `featureCallOut` was not descriptive and did not reflect the content being loaded on the index page.

This sets a consistent naming pattern (e.g. `featureDeveloperPorfolios`) for when we start adding more "features" on the home page

File: src/components/pages/home/index.tsx [hunk 3]
```diff
@@ -114,34 +117,27 @@ const Home: FunctionComponent<any> = ({homePageData}) => {
                     />
                   </div>
                   <div className="max-w-screen-sm space-y-3">
-                    <h1 className="md:text-3xl text-2xl dark:text-gray-200 font-bold leading-tight">
-                      Digital Gardening for Developers
-                    </h1>
+                    <Link href={featureDigitalGardening.path}>
```
Comment (laurosilvacom) [inline src/components/pages/home/index.tsx:120]: `digitalGardeningQuery` passes the path to the subpage `/learn/digital-gardening`

**Review by zacjones93 — COMMENTED** [review]
This looks like a great start, what do you think of adding [Digital Gardening with MDX Magic](https://egghead.io/talks/mdx-digital-gardening-with-mdx-magic) to that initial 'gardening infrastructure' list?

**Review by zacjones93 — APPROVED** [review]

**Review by tayiorbeii — APPROVED** [review]

**Pull request merged**